docs/user/_index.md

```diff
@@ -80,9 +60,30 @@ machine as the Grafana Agent process. It:
 If the filter passes, the target is allowed to be scraped. Otherwise, the target
 will be silently ignored and not scraped.
```

Comment:
not a huge fan of future tense in tech docs. Maybe 'Otherwise, the target is silently ignored and not scraped.'? Is silently an important word here? Can that adverb be removed and not alter the technical meaning?